### PR TITLE
Add chinese translation for tokenomics and staking

### PR DESCRIPTION
**Pull request by Dengjianping**
Signed-off-by: Dengjianping <djptux@gmail.com>

1. Bump docusaurus to 2.1.0
2. Add chinese translation for tokenomics and staking
3. Fix some sidebar problems

### CODE REVIEW

File: i18n/cn/docusaurus-plugin-content-docs/current/calamari/KMA/03-DistributionUpdates.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# 代币模型更新
+
+在[之前宣布](https://mantanetwork.medium.com/the-calamari-crowdloan-on-kusama-74a3cb2a2a4b) 的 KMA 代币分配计划中，我们保留了发行量10% 的 KMA 发行用于激励未来的 Calamari 众贷。而Manta团队在2022年7月成功在没有众贷的情况下为Calamari获得了第二个插槽，并且在8月成功完成了对这个新插槽的置换，没有停机或对用户造成影响。由于这个平行链插槽是自筹资金的，**9% **的** KMA **储备可以重新用于有助于提高** Calamari Network **去中心化，**1% **将用于开发基金。****
```
Comment (Dengjianping) [inline i18n/cn/docusaurus-plugin-content-docs/current/calamari/KMA/03-DistributionUpdates.md:3]: The underline is not obvious.

File: i18n/cn/docusaurus-plugin-content-docs/current/calamari/KMA/01-Overview.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- 45%进入国库，以支持未来的插槽拍卖和社区开支，如进入社区发展基金；
+- 10%分配给该区块的 Collator 节点；
+2. 作为交易媒介。
+3. 获得 Calamari 治理权。
+4. 在 Calamari 上使用 MantaPay，将公开资产与隐私资产相互转换，暂不收取额外费用。
+5. 为了进一步去中心化，在 KMA 质押上线前三年里，每年新增发行初始总量 3% KMA，奖励给质押者以保证网络去中心化。
+
+该部分所奖励的 KMA 通过铸造新的 KMA 获得，国库将定期销毁所有新铸造的 KMA，以抵消该部分产生的通胀。3年后，Calamari 社区将对未来的奖励进行重新决议。
+
+:::note
```
Comment (0xHolly) [inline i18n/cn/docusaurus-plugin-content-docs/current/calamari/KMA/01-Overview.md:18]: 注意

File: i18n/cn/docusaurus-plugin-content-docs/current/calamari/Overview.md [hunk 3]
```diff
@@ -1,16 +1,15 @@
 # 📝 概览
 
-Calamari 是 Manta Network 的社区化先行网，也是 Kusama 生态系统的链上隐私平台。使用 zkSNARK 和其它领先的密码学技术，Calamari 使链上隐私具有互操作性、易于使用、高性能和可审计性。
+[Calamari](http://www.calamari.network/) 是 Manta Network 的社区化先行网，也是 Kusama 生态系统的链上隐私平台。使用 zkSNARK 和其它领先的密码学技术，Calamari 使链上隐私具有互操作性、易于使用、高性能和可审计性。
 
 ## Calamari 平行链 
 
-Calamari 是 Manta Network 的社区化先行网，也是 Manta Network 的早期实验版本，所有新功能都将在 Calamari 上进行首发和测试。 Calamari 是为服务于 Kusama 的 DeFi 世界而构建的即插即用的平行链。它将 zkSNARK 带入 Kusama 生态，为交易和交换带来链上隐私保护。
+:::note
```
Comment (0xHolly) [inline i18n/cn/docusaurus-plugin-content-docs/current/calamari/Overview.md:7]: 请注意

**Review by 0xHolly — DISMISSED** [review]

**Comment by Dengjianping** [comment]
Followed this pr https://github.com/Manta-Network/docs/pull/98 to removed unanswered questions

**Review by stechu — APPROVED** [review]

**Review by stechu — APPROVED** [review]

**Pull request merged**